cmd/earthly/root_cmds.go

```diff
@@ -113,6 +115,12 @@ func (app *earthlyApp) rootCmds() []*cli.Command {
 			Usage:       "Earthly organization administration *experimental*",
 			Subcommands: app.orgCmds(),
 		},
+		{
+			Name:      "doc",
+			Usage:     "Document targets from an Earthfile *experimental*",
```

Comment:
Shall we launch it straight as "beta", since there is no risk of messing up existing builds?

Perhaps we also move `ls` into "beta".

Reply:
Sounds good. Does this mean that we drop `*experimental*`, or that we replace it with `*beta*`? I don't see any current subcommands marked beta.

Reply:
Good question - it seems that historically we haven't marked beta features in a specific way. I guess mostly because in beta, the API is anyway finalized. Dropping makes sense to me.